group_vars/all/vlans.yaml

```diff
@@ -1,17 +1,281 @@
+    switches:
+      - oct_data
+      - oct_mgmt
+  208:
+    name: "ESI-CONTROL-PLANE"
+    description: "ESI control plane"
+    switches:
+      - oct_data
+  209:
+    name: "OCP-PROD-PROVISIONING"
+    description: "Openshift Internal (Baremetal 4.x)"
+    switches:
+      - oct_data
+  210:
+    name: "MOC-NFS-ZERO-CLUSTER"
+    description: "NFS for zero cluster"
+    switches:
+      - oct_data
+  211:
+    name: "MOC-NESE"
+    description: "New England Storage Exchange (NESE)"
+    switches:
+      - oct_data
+  212:
+    name: "OCP-STAGING-INTERNAL"
+    description: "Openshift Staging Internal (ocp-staging)"
+    switches:
+  213:
+    name: "MOC-PROD-CEPH-ISCSI"
+    description: "Ceph Cluster iSCSI"
+    switches:
+      - oct_data
+  249:
+    name: "CEPH-INTERNAL"
+    description: "Ceph Cluster (internal)"
+    switches:
+  250:
+    name: "MOC-PROD-CEPH"
+    description: "Ceph public (for clients)"
+    switches:
+      - oct_data
+  252:
+    name: "MOC-RESEARCH-CEPH"
+    description: "****"
+    switches:
+      - oct_data
+  259:
+    name: "STAGING-FOREMAN"
+    description: "Staging - Foreman"
+    switches:
+  267:
+    name: "MOC-RESEARCH-CEPH-EXTRA"
+    description: "****"
+    switches:
+      - oct_data
+  270:
+    name: "STAGING-INTERNAL"
+    description: "Staging - Internal API"
+    switches:
+  271:
+    name: "STAGING-TENANT"
+    description: "Staging - Tenant Network"
+    switches:
+  272:
+    name: "STAGING-PUBLIC"
+    description: "Staging - Public Network"
+    switches:
+  273:
+    name: "STAGING-OSSTACK"
+    description: "Staging - OS Stack isolation for trunk only ports"
+    switches:
+  277:
+    name: "MOC-MAAS"
+    description: "****"
+    switches:
+      - oct_data
+  280:
+    name: "MOC-MAAS-NIC1"
+    description: "****"
+    switches:
+      - oct_data
+  300:
+    name: "UNITY-INTERNAL"
+    description: "****"
+    switches:
+      - oct_data
+  301:
+    name: "UNITY-MGMT"
+    description: "****"
+    switches:
+      - oct_data
+  308:
+    name: "MOC-JIANZZHA"
+    description: "****"
+    switches:
+      - oct_data
+  500:
+    name: "MOC_BMI"
+    description: "****"
+    switches:
+      - oct_data
+  501:
+    name: "MOC-BMI"
+    description: "****"
+    switches:
+      - oct_data
+  701:
+    name: "ENGAGE1-FOREMAN"
+    description: "Foreman Provisioning. SNMP for OpenStack and Ceph	"
+    switches:
+  702:
+    name: "ENGAGE1-OPENSTACK-INTERNAL"
+    description: "OpenStack internal API"
+    switches:
+  703:
+    name: "ENGAGE1-OPENSTACK-TENANT"
+    description: "OpenStack tenant network"
+    switches:
+  704:
+    name: "ENGAGE1-INTRANET"
+    description: "Intranet (routable to internet). SNMP for client nodes."
+    switches:
+  749:
+    name: "ENGAGE1-CEPH"
+    description: "Ceph Cluster (internal)"
+    switches:
+  750:
+    name: "ENGAGE1-CEPH-PUBLIC"
+    description: "Ceph public (for clients)"
+    switches:
```

Comment:
I think besides the Engage1 Foreman network, all other engage1 networks are kind of not in use since all that hardware is shut down. But we can deal with this later.